app/controllers/claimToAdjustPoa/PoaAdjustedController.scala

```diff
@@ -66,12 +69,17 @@ class PaymentsOnAccountAdjustedController @Inject()(val authorisedFunctions: Aut
     }
   }
 
-  private def setJourneyCompletedFlag(isAgent: Boolean, poa: PaymentOnAccountViewModel)(implicit user: MtdItUser[_]): Future[Result] = {
+  private def setJourneyCompletedFlag(isAgent: Boolean, poa: PaymentOnAccountViewModel, session: PoAAmendmentData)(implicit user: MtdItUser[_]): Future[Result] = {
```

Comment:
Not part of this ticket but I feel like this method shouldn't be delivering the action here

Reply:
Because of the EitherT stuff there isn't a neat way to move the view call back into show without ruining the point of putting this logic in another method. Have renamed the method